### PR TITLE
Enable PMD plugin

### LINKED ISSUE #19: Enable PMD and fix any violations

### PR DESCRIPTION
**Pull request by meltsufin**
Fixes #19.
For now, only the default rulesets will be applied.

### CODE REVIEW

**Comment by loosebazooka** [comment]
have you got a link I can reference for this?

**Comment by meltsufin** [comment]
See: https://maven.apache.org/plugins/maven-pmd-plugin/examples/violationChecking.html

File: pom.xml [hunk 1]
```diff
@@ -255,6 +255,29 @@
...
+      <!-- PMD -->
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-pmd-plugin</artifactId>
+        <version>3.6</version>
+        <configuration>
+          <linkXRef>false</linkXRef>
+          <printFailingErrors>true</printFailingErrors>
+          <excludes>
+            <exclude>**/HelpMojo.java</exclude>
```
Comment (loosebazooka) [inline pom.xml]: I don't see this file anywhere?

Comment (meltsufin) [inline pom.xml]: It's automatically generated and can be found in target/generated-sources.

**Comment by loosebazooka** [comment]
:ship:

**Pull request merged**